### PR TITLE
update JSqueeze to version 2.0.4

### PR DESCRIPTION
**Pull request by georgehrke**
please review @DeepDiver1975 @PVince81

### CODE REVIEW

File: composer/autoload_psr4.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
     'Sabre\\CalDAV\\' => array($vendorDir . '/sabre/dav/lib/CalDAV'),
     'React\\Promise\\' => array($vendorDir . '/react/promise/src'),
     'Punic\\' => array($vendorDir . '/punic/punic/code'),
-    'Patchwork\\' => array($vendorDir . '/patchwork/jsqueeze/src', $vendorDir . '/patchwork/utf8/src/Patchwork'),
+    'Patchwork\\' => array($vendorDir . '/patchwork/utf8/src/Patchwork', $vendorDir . '/patchwork/jsqueeze/src'),
```
Comment (georgehrke) [inline composer/autoload_psr4.php]: Is this all right?

Comment (DeepDiver1975) [inline composer/autoload_psr4.php]: yes - just those two elements are flipped

**Comment by LukasReschke** [comment]
Needs rebase.

**Comment by georgehrke** [comment]
done

**Comment by DeepDiver1975** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
@georgehrke please open a pr in core which updates the submodule - THX

**Comment by jknockaert** [comment]
I don't get the point of this PR. As mentioned in https://github.com/owncloud/calendar/issues/205, the update to 2.0.4 doesn't fix the original issue. First https://github.com/tchwork/jsqueeze/issues/43 needs to be fixed. We risk breaking something else in the meantime.

**Comment by georgehrke** [comment]
@jknockaert It fixes the original issue, but there is yet another bug in JSqueeze, which causes the color issue. Though this is not as severe as the one fixed by this pr.

**Comment by jknockaert** [comment]
@georgehrke OK. That means that https://github.com/owncloud/calendar/pull/268 can be reversed now. I'll report it in https://github.com/owncloud/calendar/issues/205.

**Comment by georgehrke** [comment]
@jknockaert It can be reversed once this fix made it into the stable version of ownCloud 9 :)

**Pull request merged**